src/tex/ms.tex

```diff
@@ -267,7 +267,7 @@ \subsection{Curation workflow}\label{sec:curation}
 
 \begin{table}[!h]
     \centering
-    \caption{\textbf{Overview of sources of the curated questions}. The table provides an overview of the types of sources the questions have been curated from. Detailed sources are available in the source data on GitHub. Questions without source have been curated completely from scratch. Questions based on lecture notes or URL have been curated based on content presented in those resources. All questions have been rephrased, annotated, and reviewed before being added to the corpus.}
+    \caption{\textbf{Overview of sources of the curated questions}. The table provides an overview of the types of sources the questions have been curated from. Detailed sources are available in the source data on GitHub. Questions without source have been curated completely from scratch. Questions based on lecture notes or URL have been curated based on content presented in those resources. All questions have been rephrased, annotated, and reviewed before being added to the corpus. Note that the chemical preference questions are the main corpus of the semi-automatically generated group with 1001 questions of the total 1752.}
```

Comment:
If we have numbers, it would be amazing to have them included via `\variable` to ensure that they are automatically updated when we change something upstream

Reply:
If you don't know how I can also take care of it

Reply:
I'll try